### PR TITLE
Doxygen fixes

### PR DESCRIPTION
**Pull request by VictorLamoine**

### CODE REVIEW

File: common/include/pcl/common/time.h [hunk 1]
```diff
@@ -155,8 +155,9 @@ namespace pcl
 
       /** \brief Constructor.
         *
-        * \param[i] window_size number of most recent events that are
-        * considered in frequency estimation (default: 30) */
+        * \param[in] window_size number of most recent events that are
+        * considered in frequency estimation (default: 30)
+        * \return Event frequency */
```
Comment (taketwo) [inline common/include/pcl/common/time.h]: Constructors don't return... or do they?

Comment (VictorLamoine) [inline common/include/pcl/common/time.h]: Didn't see it was a constructor, it fixes the doxygen warning though..

Comment (taketwo) [inline common/include/pcl/common/time.h]: Do you mean that _doxygen_ throws warnings for constructors without `\return`? Weird...

Comment (VictorLamoine) [inline common/include/pcl/common/time.h]: Checking this right now

Comment (VictorLamoine) [inline common/include/pcl/common/time.h]: I must be crazy, there's no warning.

**Pull request merged**